airflow/contrib/hooks/cassandra_hook.py

```diff
@@ -158,6 +158,21 @@ def get_lb_policy(policy_name, policy_args):
                                                            child_policy_args)
                 return TokenAwarePolicy(child_policy)
 
+    def table_exists(self, table):
+        """
+        Checks if a table exists in Cassandra
+
+        :param table: Target Cassandra table.
+                      Use dot notation to target a specific keyspace.
+        :type table: string
+        """
+        keyspace = None
+        if '.' in table:
+            keyspace, table = table.split('.', 1)
+        m = self.get_conn().cluster.metadata
+        return (keyspace in m.keyspaces and
```

Comment:
If the keyspace is `None`, then this predicate will always evaluate to `False`, right?

Reply:
Yeah, that's a mistake. `self.keyspace` should be used if a keyspace is not specified in CQL.